Handle pelican image field

QuintenQVD0 opened the pull request:
# Changes

- When converting from pelican to ptero remove the image field
- When converting from ptero to pelican add null to the image field

QuintenQVD0 committed:
Remove image property from ptero object

QuintenQVD0 committed:
Add image property to pelican metadata and order

redthirten reviewed (changes requested):
Looks good, but we need to add a field to the optional metadata section (for converting to Pelican) that allows the user to specify the icon image (via file or value). For example, this would be important if a Ptero user is trying to update an existing Pelican Egg in the Eggs repo. Just like with UUID, they need to copy the value from the Pelican Egg into the Scramble form so that Pelican Egg output does not cause extra diffs (eg. null UUID or image keys) in the PR.

Unless you want to work on this, I can add this tomorrow (or maybe tonight) to this PR. We can also just add this later if merging this PR is super urgent.

QuintenQVD0 commented:
I will let you do that if you don't mind

redthirten committed:
Add Image field to optional metadata section

- Added a field to optionally specify a base 64 image to be added as metadata to the Egg when converting to Pelican.
- Minor HTML and JS refactoring.
- Small README clarification.
- Bumped version in package.json
